backend/tests/conftest.py

```diff
@@ -147,27 +163,157 @@ def prefect_test(prefect_test_fixture):
         yield
 
 
+@pytest.fixture(scope="session")
+def neo4j(request: pytest.FixtureRequest, load_settings_before_session) -> Optional[DockerContainer]:
+    if not INFRAHUB_USE_TEST_CONTAINERS or os.getenv("INFRAHUB_DB_TYPE") == "memgraph":
```

Comment:
I think we could convert both `INFRAHUB_USE_TEST_CONTAINERS`  into a proper fixture, we need to ensure we are converting properly what we are getting from the env vars into a boolean

for INFRAHUB_DB_TYPE, could we read that value directly from the config file `config.SETTINGS.database.db_type` ?

Reply:
Right, I used `str_to_bool` to properly convert env variable into a boolean.